codec-http/src/test/java/io/netty/handler/codec/http/websocketx/extensions/compression/PerMessageDeflateEncoderTest.java

```diff
@@ -273,15 +282,14 @@ public void testIllegalStateWhenCompressionInProgress() {
     public void testEmptyFrameCompression() {
         EmbeddedChannel encoderChannel = new EmbeddedChannel(new PerMessageDeflateEncoder(9, 15, false));
 
-        TextWebSocketFrame emptyFrame = new TextWebSocketFrame("");
+        TextWebSocketFrame emptyFrame = new TextWebSocketFrame(encoderChannel.bufferAllocator(), "");
 
         assertTrue(encoderChannel.writeOutbound(emptyFrame));
         TextWebSocketFrame emptyDeflateFrame = encoderChannel.readOutbound();
 
         assertEquals(WebSocketExtension.RSV1, emptyDeflateFrame.rsv());
-        assertTrue(ByteBufUtil.equals(EMPTY_DEFLATE_BLOCK, emptyDeflateFrame.content()));
-        // Unreleasable buffer
-        assertFalse(emptyDeflateFrame.release());
+        assertEquals(EMPTY_DEFLATE_BLOCK.get(), emptyDeflateFrame.binaryData());
```

Comment:
How about checking the contents of the `emptyDeflateFrame` instead? The buffer we "get" is not closed.